CHANGES.md

```diff
@@ -1,6 +1,8 @@
 Unreleased
 ----------
 
+- coqdep is now called once per theory (#7048, @Alizter)
```

Comment:
Maybe note this should imply a speedup ? Maybe the wording "instead of once per file. This should make Coq builds noticeable faster as coqdep startup time is non-trivial"

Also you may want to add `ejgallego` here if you need.

Reply:
Yes, my mistake.